enumfields/enums.py

```diff
@@ -114,6 +115,10 @@ def class_to_str(cls):
 
 
 class ChoicesEnum(enum.Enum, metaclass=ChoiceEnumMeta):
+    value: Any
+    label: str
+    initial: bool
+    next: list[str] | None
```

Comment:
next should be set(str) | None. Also label should be StrPromise

Reply:
for the label, maybe consider use of StrOrPromise. We might have some occurrences of untranslated labels.

Reply:
I did not use StrOrPromise by choice. I think it is unnecessary to include another dependency only for this type. Mypy won't ever compare the real value passed to `Choice` constructor with this type so we can safely use `str`.